amundsen_application/static/js/components/TableDetail/TableIssues/styles.scss

```diff
@@ -27,12 +23,49 @@
     white-space: nowrap;
     overflow: hidden;
     text-overflow: ellipsis;
+    max-width: fit-content;
   }
   .issue-title-display-text {
     display: flex;
     max-width: fit-content;
   }
-  .table-issue-more-issues {
-    margin: 0px 2px 0px 2px; 
+  .table-issue-status {
+    display: flex;
+    justify-content: flex-end;
+    flex-grow: 1.1;
+    min-width: fit-content;
+    margin: 0px 0px 0px 5px;
+  }
+  .table-issue-priority {
+    margin: 0px 5px 0px 0px;
+    border: 1px solid $rose80; 
+    border-radius: 5px;
+    padding: 1px 2px;
+    flex: none; 
+  }
+  .blocker {
```

Comment:
Are these kind of styles something that should be configurable? I wasn't sure how to properly handle configuration here since the "priorities" are set by the issue tracker rather than by the application.

Reply:
Yeah, the colors came straight from the recommendations from the LDL - I just followed what the designs had + recommendations on opacity of color.